supabase/seed.sql

```diff
@@ -12,15 +12,64 @@
 -- | user.8@example.com  | User.8.Password  |
 -- | user.9@example.com  | User.9.Password  |
 -- | user.10@example.com | User.10.Password |
+-- | landlord@example.com|      123456      |
+-- |landlord1@example.com|      123456      |
```

Comment:
Is there a reason none of the existing user accounts could have been used here??

If you are going to create new accounts, either because your new tests would interfere with existing ones, can we try to stick to a similar naming scheme (e.g.` landlord.1@example.com` & `Landlord.1.Password`). You will also then need to expand the auth setup in `playwright-tests/auth.setup.ts` to login these user's if we want to be able to do any tests logged in as them.